src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketCreate.java

```diff
@@ -192,11 +193,29 @@ public boolean failIfExists() {
 
     void validate(FailureCollector collector) {
       if (!containsMacro("paths")) {
+        Storage storage = null;
+        try {
+          Credentials credentials = getServiceAccountFilePath() == null ?
+            null : GCPUtils.loadServiceAccountCredentials(getServiceAccountFilePath());
+          storage = GCPUtils.getStorage(getProject(), credentials);
+        } catch (IOException e) {
+          collector.addFailure(e.getMessage(), "Ensure you entered the correct file path.")
```

Comment:
what kind of message is displayed ? Is `IOException` only thrown when path is not correct?

Reply:
In this case, it should be some kind of "File Not Found" error or, in cases where it is unable to open the file for some reason, it will display that. However, this is outdated since we are moving these checks to happen only at runtime.